articles/active-directory/manage-apps/configure-user-consent.md

```diff
@@ -139,10 +139,54 @@ You can use the Azure AD PowerShell Preview module ([AzureADPreview](https://doc
     }
     ```
 
+## Configure risk based step-up consent
+
+Risk based step-up consent helps reduce user exposure to malicious apps making [illicit consent requests](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants.md). If Microsoft detects a risky end-user consent request, the request will require a "step-up" to admin consent instead. This capability is enabled by default, but it will only result in a behavior change when end-user consent is enabled.
```

Comment:
```suggestion
Risk-based step-up consent helps reduce user exposure to malicious apps making [illicit consent requests](https://docs.microsoft.com/microsoft-365/security/office-365-security/detect-and-remediate-illicit-consent-grants). If Microsoft detects a risky end-user consent request, the request will require a "step-up" to admin consent instead. This capability is enabled by default, but it will only result in a behavior change when end-user consent is enabled.
```